src/generators/genlua.ml

```diff
@@ -477,6 +525,10 @@ and gen_expr ?(local=true) ctx e = begin
 		spr ctx "]";
 	| TBinop (op,e1,e2) ->
 		gen_tbinop ctx op e1 e2;
+	| TField ({eexpr = TCall _} as e, f) when is_multireturn ctx e ->
```

Comment:
If we're immediately extracting a multireturn value from a method call that generates one, we use the "select" method since it doesn't require creating any temp variables.